src/libstd/primitive_docs.rs

```diff
@@ -67,6 +67,123 @@
 #[stable(feature = "rust1", since = "1.0.0")]
 mod prim_bool { }
 
+#[doc(primitive = "never")]
+//
+/// The `!` type, also called "never".
```

Comment:
This "introduction" sounds strange. I expected to have something after directly but just a dot. So strange.

Reply:
This kind of introduction line matches the other primitives. Compare with "Raw, unsafe pointers, `*const T`, and `*mut T`", "References, both shared and mutable", "A finite heterogeneous sequence, `(T, U, ..)`", all of which appear in the current docs. Do you have a better suggestion?